zdm/grid.py

```diff
@@ -41,6 +41,9 @@ def __init__(self, survey, state,
         self.b_fractions=None
         # State
         self.state = state
+        
+        # FOR TESTING WITH LOG 10
```

Comment:
Is this comment still accurate?

Reply:
In particular, shouldn't the use_log10 be passed through the 'state'? Or is this now just a speedup which should work and hence we hard-code it?

Reply:
fixed